File: HB/pysrc/프로그래머스/레벨0/Day15/영어가_싫어요.py
def solution(numbers):
    answer = 0
    a=['zero','one', 'two','three','four', 'five', 'six','seven','eight','nine']
    while len(numbers):
        if (numbers[0:3]) in a:
            answer=answer*10+a.index(numbers[0:3])
            numbers=numbers[3:]
        elif (numbers[0:4]) in a:
            answer=answer*10+a.index(numbers[0:4])
            numbers=numbers[4:]
        elif (numbers[0:5]) in a:
            answer=answer*10+a.index(numbers[0:5])
            numbers=numbers[5:]
    return answer

File: SB/src/프로그래머스/레벨0/Day22/영어가_싫어요.java
package 프로그래머스.레벨0.Day22;

public class 영어가_싫어요 {
    class Solution {
        public long solution(String numbers) {
            // numbers = numbers.replaceAll("zero","0");
            // numbers = numbers.replace("one","1");
            // numbers = numbers.replace("two","2");
            // numbers = numbers.replace("three","3");
            // numbers = numbers.replace("four","4");
            // numbers = numbers.replace("five","5");
            // numbers = numbers.replace("six","6");
            // numbers = numbers.replace("seven","7");
            // numbers = numbers.replace("eight","8");
            // numbers = numbers.replace("nine","9");
            String arr[] = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};

            for (int i = 0; i < arr.length; i++) {
                numbers = numbers.replaceAll(arr[i], String.valueOf(i));
            }

            long answer = Long.parseLong(numbers);
            return answer;
        }
    }
}
